views/jira-redirect.hbs

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
```

Comment:
This is a temporary page to complete the flow. Actual page will be developed as part of another ticket.